### ISSUE TITLE
Tooltips in WYSIWYG are not being translated

### ISSUE THREAD

**Issue by Posoroko** [directus/directus#18627]
### Describe the Bug

The default language of my project is set to French, but the tooltips for the buttons in the wisiwig fields are still in English. 

### To Reproduce

Set the default language to French

### Directus Version

10.0

### Hosting Strategy

Self-Hosted (Docker Image)

**Comment by Posoroko** [comment]
I'm not sure if I needed to do anything after the issue was added to linear.  I still haven't find the solution.  Since then, I added the button to create a link in the WYSIWYG field and this new button is being translated to French.   The other ones aren't though. I tried removing them and putting them back, but still no translation.
I'm not really used to issues on GitHub, I would like to be as helpful as possible. :-)

**Comment by rijkvanzanten** [comment]
@Posoroko Linear is a way for the core team to keep a better overview of all the issue that exist across all of our repos, nothing more 🙂 

The real bug here is that tinymce uses `title` attributes on those buttons for the labels, which means that it effectively skips over Directus' translations completely. My ideal solution would be for those buttons to use proper Directus tooltips, but that might be very hacky to support 🤔 

As a "good enough" middleground, we might be able to dynamically set tinymce's `language` option (https://www.tiny.cloud/docs/configure/localization/#language) to match the user's language 🤔

**Comment by Posoroko** [comment]
>As a "good enough" middleground, we might be able to dynamically set tinymce's `language` option (https://www.tiny.cloud/docs/configure/localization/#language) to match the user's language 🤔

ok, I see.  Have you heard of anyone achieving this before?  Tinymce's doc says we need to download the needed languages.  Would there be a way to download French and make it available to Directus?

This is such a small problem, I'm sorry about bothering you with it!

**Comment by Posoroko** [comment]
> We might be able to use that same logic to dynamically load in the tinymce locale file and register it in the instance

I wish I could help you guys out with that. But I'm too much of a noob to contribute to such a big project. maybe some day :-)

Do you think it's something that you might implement?  I work in France so all my directus projects are in French.   Tooltip being the only way to know what a button does, it could be a quite usefull functionality!

**Comment by rijkvanzanten** [comment]
> Do you think it's something that you might implement? I work in France so all my directus projects are in French. Tooltip being the only way to know what a button does, it could be a quite usefull functionality!

Yeah! .. eventually 😬 Seeing this is a pretty small bug that doesn't affect a lot of users, it's hard to prioritize it over some of the more pressing issues 🤔
